pallets/lbp/src/benchmarking.rs

```diff
@@ -227,7 +227,7 @@ benchmarks! {
 		ensure!(PoolData::<T>::contains_key(&pool_id), "Pool does not exist.");
 
 		let pool_data = LBP::<T>::pool_data(&pool_id);
-		assert_eq!(pool_data.paused, false);
+		assert!(!pool_data.paused);
```

Comment:
i don't like double negation 🙃

Reply:
and i think that `assert_eq` is more appropriate in this case anyway

Reply:
There is only one negation :) I agree with Clippy (Clippy suggests to use assert macro). Using `assert_eq` is like using `if condition == true` anti-pattern.

Reply:
ok, you're right, but the second `!` is hard to spot, I actually had to read it multiple times what you';re trying to check, before it was much more obvious

Reply:
i am not looking at it as predicate, but simply as value check - that's why actually asset_eq makes more sense to me